### PR TITLE
[mono] Add Vector128 ConvertToInt32 intrinsics for Amd64

### PR DESCRIPTION
**Pull request by matouskozak**
Add support for the following Vector128 API's:

- ConvertToInt32 (no ConvertToUInt32 version yet)

### CODE REVIEW

File: src/mono/mono/mini/simd-intrinsics.c [hunk 1]
```diff
@@ -1107,6 +1107,17 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 #endif
 	}
 	case SN_ConvertToInt32: 
+#ifdef TARGET_AMD64
```
Comment (vargaz) [inline src/mono/mono/mini/simd-intrinsics.c:1110]: This looks a bit weird, this code should be in the original case block inside an if (id == SN_ConvertToInt32).

**Review by vargaz — APPROVED** [review]

**Comment by matouskozak** [comment]
Replaced by https://github.com/dotnet/runtime/pull/77211 thus closing this PR.

**Pull request closed without merging**